modin/pandas/base.py

```diff
@@ -11,7 +11,34 @@
 # ANY KIND, either express or implied. See the License for the specific language
 # governing permissions and limitations under the License.
 
-"""Implement DataFrame/Series public API as pandas does."""
+"""
+Implement DataFrame/Series public API as pandas does.
+
+Methods docstrings contain tables with supported parameters information.
+These tables are structed as follows: The first column contains the parameter name,
+the second, third and fourth column - flags describing particular properties of
+method parameters for a concrete execution supported by Modin (`PandasOnRay`, `PandasOnDask`
+or `OmniSci`), and the last one is for parameter notes. The flags stand for the following:
+
++-----------+-----------------------------------------------------------------------------------------------+
+| Flag      | Meaning                                                                                       |
++===========+===============================================================================================+
+| Harmful   | Usage of this parameter can be harmful for performance of your application. Usually this      |
+|           | happens when parameter (full range of values and all types) is not supported and default      |
```

Comment:
I would rephrase the part of this sentence about defaulting to pandas like `... is not supported and we are (or Modin is) defaulting to pandas`. Also, we should probably put a link to defaulting to pandas page https://modin.readthedocs.io/en/stable/supported_apis/defaulting_to_pandas.html.

Reply:
Ok, rephrased to `... is not supported and Modin defaulting to pandas` and link is added.